sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -650,23 +672,30 @@ public void run() {
                         iterator.remove();
                         removeAllStoredErrorLogFiles(id);
                     }
-                } else {
+                }
+
+                /* We send the crash. */
+                else {
 
+                    /* Always send: we remember. */
                     if (userConfirmation == ALWAYS_SEND) {
                         StorageHelper.PreferencesStorage.putBoolean(PREF_KEY_ALWAYS_SEND, true);
                     }
 
+                    /* Send every pending report. */
                     Iterator<Map.Entry<UUID, ErrorLogReport>> unprocessedIterator = mUnprocessedErrorReports.entrySet().iterator();
                     while (unprocessedIterator.hasNext()) {
-                        if (shouldStopProcessingPendingErrors()) {
-                            break;
-                        }
+
+                        /* Send report. */
                         Map.Entry<UUID, ErrorLogReport> unprocessedEntry = unprocessedIterator.next();
                         ErrorLogReport errorLogReport = unprocessedEntry.getValue();
                         mChannel.enqueue(errorLogReport.log, ERROR_GROUP);
 
-                        Iterable<ErrorAttachmentLog> attachments = mCrashesListener.getErrorAttachments(errorLogReport.report);
-                        handleErrorAttachmentLogs(attachments, errorLogReport);
+                        /* Get attachments from callback in automatic processing. */
+                        if (mAutomaticProcessing) {
```

Comment:
Do you need to check here? Shouldn't it process error attachments?

Reply:
No it works differently in React Native: https://github.com/Microsoft/mobile-center-sdk-react-native/blob/9de1aa7d0edb00fd2a2ee3bfc320c2b3971a9d39/mobile-center-crashes/Crashes.js#L104

Reply:
Is `mAutomaticProcess` only for RN or it is also used by Xamarin?

Reply:
Xamarin uses automatic processing and is not impacted by these changes.

Reply:
However we expect that Unity and Cordova would use it in the future if and when we will support crashes for those.